.drone.yml

```diff
@@ -83,84 +93,129 @@ pipeline:
+      KUBE_DEPLOYMENT:
+        from_secret: KUBE_DEPLOYMENT
+      KUBE_NAMESPACE:
+        from_secret: KUBE_NAMESPACE
+      KUBE_SERVER:
+        from_secret: KUBE_SERVER
+      KUBE_TOKEN:
+        from_secret: KUBE_TOKEN
     when:
-      branch: master
-      event: push
+      branch:
+        - master
+      event:
+        - push
 
-  e2e_test:
-    image: quay.io/ukhomeofficedigital/lev-ci
-    secrets:
-      - DEV_URL
-      - DEV_USERNAME
-      - DEV_PASSWORD
-    environment:
-      - DOCKER_HOST=tcp://172.17.0.1:2375
-      - DOCKER_REPO=quay.io
-      - DOCKER_BASEDIR=/ukhomeofficedigital/
-      - DOCKER_IMAGE=lev-web-e2e
+  - name: e2e_test
+    pull: if-not-exists
+    image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
     commands:
+      - sleep 20
       - PACKAGE_VERSION=$(grep version package.json | head -1 | awk -F ":" '{ print $2 }' | sed 's/[",]//g' | tr -d '[[:space:]]')
       - TAG="v$${PACKAGE_VERSION}-$${DRONE_BUILD_NUMBER}-$${DRONE_COMMIT}"
       - IMAGE="$${DOCKER_REPO}$${DOCKER_BASEDIR}$${DOCKER_IMAGE}"
       - set -o pipefail
       - docker run --rm --name "$${DOCKER_IMAGE}" -e "ENV=dev" -e "TEST_URL=$${DEV_URL}" -e "USERNAME=$${DEV_USERNAME}" -e "PASSWORD=$${DEV_PASSWORD}" "$${IMAGE}:$${TAG}" | tee 'test.log'
+    environment:
+      DEV_PASSWORD:
+        from_secret: DEV_PASSWORD
+      DEV_URL:
+        from_secret: DEV_URL
+      DEV_USERNAME:
+        from_secret: DEV_USERNAME
+      DOCKER_BASEDIR: /ukhomeofficedigital/
+      DOCKER_HOST: tcp://127.0.0.1:2375
+      DOCKER_IMAGE: lev-web-e2e
+      DOCKER_REPO: quay.io
     when:
-      branch: master
-      event: push
+      branch:
+        - master
+      event:
+        - push
 
-  save_test_logs:
+  - name: save_test_logs
+    pull: if-not-exists
     image: quay.io/ukhomeofficedigital/kd:latest
-    secrets:
-      - KUBE_SERVER
-      - KUBE_NAMESPACE
-      - KUBE_TOKEN
-      - KUBE_DEPLOYMENT
     commands:
       - kubectl="kubectl --insecure-skip-tls-verify --server=$${KUBE_SERVER} --namespace=$${KUBE_NAMESPACE} --token=$${KUBE_TOKEN}"
       - $${kubectl} annotate --overwrite deployment $${KUBE_DEPLOYMENT} lev-web-test-log="`cat test.log`"
+    environment:
+      KUBE_DEPLOYMENT:
+        from_secret: KUBE_DEPLOYMENT
+      KUBE_NAMESPACE:
+        from_secret: KUBE_NAMESPACE
+      KUBE_SERVER:
+        from_secret: KUBE_SERVER
+      KUBE_TOKEN:
+        from_secret: KUBE_TOKEN
     when:
-      branch: master
-      event: push
+      branch:
+        - master
+      event:
+        - push
 
-  promote_to_uat:
+  - name: promote_to_uat
+    pull: if-not-exists
     image: quay.io/ukhomeofficedigital/drone-trigger:latest
-    secrets: [ drone_token ]
-    drone_server: https://drone-gitlab.acp.homeoffice.gov.uk
-    repo: lev/levops
-    branch: master
-    event: push
-    deploy_to: uat
-    when:
+    settings:
       branch: master
+      deploy_to: uat
+      drone_server: https://drone-gitlab.acp.homeoffice.gov.uk
```

Comment:
url need to point to new drone

Reply:
Updated to https://drone-gh.acp.homeoffice.gov.uk/